image/manifest.go

```diff
@@ -240,3 +240,33 @@ loop:
 	}
 	return nil
 }
+
+// ValidateManifestMediaType validate the manifest schema media-type
+// and its descendants fields media type, such as config and layers,
+// to make sure they match to spec definition, or returns an error if
+// the validation failed.
+func ValidateManifestMediaType(r io.Reader) error {
```

Comment:
With the call to `schema.MediaTypeManifest.Validate` at the end of this function, the name should probably just be `ValidateManifest` (since it covers descriptor media types locally *and* the JSON Schema via `schema.MediaTypeManifest.Validate`.